docs/release_notes/0.75.0.md

```diff
@@ -0,0 +1,18 @@
+# Release 0.75.0
+
+
+## Features
+
+- Add --config flag to get labels & nodegroups (#4469) (#4465)
+- Add option to skip endpoint creation completely (#4457)
```

Comment:
This might not be clear without mentioning private clusters. I think we should rephrase it to:

```suggestion
- Add option to skip endpoint creation for private clusters (#4457)
```